x-pack/plugins/cases/public/common/navigation/deep_links.ts

```diff
@@ -49,3 +49,36 @@ export const getCasesDeepLinks = <T extends AppDeepLink = AppDeepLink>({
     },
   ],
 });
+
+export const getCasesLinkItems = ({
```

Comment:
These 2 functions are almost the same, wouldn't it be possible to call `getCasesDeepLinks` from our side instead, and convert the result to our LInkItem type there?
This way we don't have to deal with the `LinkItem` type (Security Solution specific) inside Cases.